<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>922</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestor de Personal</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/usuario.png</normaloff>../../../Downloads/usuario.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>301</width>
      <height>431</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Microsoft Yi Baiti</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Datos de Registro</string>
    </property>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>271</width>
       <height>89</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="boton_nuevo">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nuevo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boton_borrar">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Borrar</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="boton_modificar">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Modificar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>64</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label_nombre">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Nombres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_apellido">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Apellidos</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_sexo">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Sexo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_edad">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Edad</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_empleo">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Empleo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_numero">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Telefono</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_email">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Email</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>191</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="line_nombre">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_apellido">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_sexo">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_edad">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_empleo">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_telefono">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="line_email">
        <property name="font">
         <font>
          <family>Microsoft Sans Serif</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QTableView" name="tabla">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>561</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>922</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
